airbyte-integrations/connectors/source-outbrain-amplify/metadata.yaml

```diff
@@ -0,0 +1,20 @@
+data:
+  allowedHosts:
+    hosts:
+      - TODO # Please change to the hostname of the source.
+  registries:
+    oss:
+      enabled: false
```

Comment:
This should be true

Reply:
Also can you enable this connector for cloud by adding the cloud registry value and setting it to true?